docs/server/introduction.mdx

```diff
@@ -3,22 +3,68 @@ title: Server SDKs
 
-Or, you can jump right in to the technical docs, by language, on the sidebar or below:
+- The `getConfig` call takes a [**StatsigUser**](/sever/concepts/user) object with a user identifier and returns the dynamic configuration based on the rules that you've configured for your end-users; use the returned `config` object to `get` the value of the parameters required to implement a dynamically configured app experience
+
+- The server SDK automatically flushes all accumulated exposure checks to the Statsig servers every 60 seconds
+
+- You can verify that these exposure events are being recorded in Statsig servers using the live **Exposure Stream** in the Statsig console under the **Diagnostics** tab of your experiment
+ 
+-	The **Users** and **Checks** charts in the **Diagnostics** tab are updated hourly
+
+#### Updating Experiment Configuration
+
+- The server SDK polls Statsig servers for updates to the rule sets every 10 seconds. Statsig’s servers service these requests through a cache with a TTL of 60 seconds. This means that it may take 10 - 70 seconds to propagate any rule updates to the server SDK. 
+
+
+:::info Best Practices
+
+**Why is StatsigUser with a userID required for server SDKs?**
+
+Learn about [UserID Requirements](/messages/serverRequiredUserID) for the [StatsigUser](/server/concepts/user) while using a server SDK.
+
+**Setting Default Parameter Values**
+
+Statsig recommends including a default value when you make the `get` call to fetch a parameter value from the returned `config` object. This will ensure that your application code falls back to the default value for a parameter that's not yet been configured in an experiment, layer, or dynamic config in the Statsig console. For convenience, Statsig by default returns _false_ when you check a gate that has not yet been configured.   
+
+:::
+
+:::info Testing exposure checks in development or staging environments
+
+You can test your experiment in development or staging environments by setting the environment tier in the `initialize` call. When you do this, you can verify that your exposure checks are working by switching the toggle to _Show non-production logs_ in the **Exposure Stream** under the **Diagnostics** tab for your experiment in the Statsig console. By default, the **Exposure Stream** shows exposure checks logged in production environments. 
+
+When you're testing in development or staging environments, you can also target specific members in your team to see a specific variant by adding these members to the override list of a rule or variant group using the **Manage Overrides** option.  
+
+:::
+
+### 3. Logging Custom Events
+
+-	The server SDK's `logEvent` call takes the **server SDK key** with a user identifier along with a custom event that you want to record to analyze the impact of the experiment on your end-user experience
+
+-	The server SDK automatically flushes all accumulated custom checks to the Statsig servers every 60 seconds
```

Comment:
There is also shutdown() for server SDKS, in case you are using a script or operating in a serverless environment and want to force logs to flush